### DISCUSSION TITLE
I have problem with tteck script

### DISCUSSION THREAD

**Discussion by shwanaa** [tteck/Proxmox#1818] [Help Needed] [answered]
![Screenshot_2023-09-20-22-00-30-007_com.kenfenheuer.proxmoxclient.jpg](https://github.com/tteck/Proxmox/assets/54441031/f6c2765d-2adb-4b2a-bafd-4fed0d2d9d01)

Previously, when I entered the script link, it would download immediately

But now it goes back to the same text

**Comment by tteck** [comment, marked as answer]
It seems that you're facing a network problem, _probably_ linked to DNS.

**Reply by shwanaa** [reply]
I think it's my Proxmox fault

**Reply by shwanaa** [reply]
You were right it was
